addon/helpers/list.js

```diff
@@ -1,6 +1,16 @@
 import { MicroState } from 'ember-microstates';
+import assign from '../utils/assign';
 
 export default MicroState.extend({
+
+  wrap(value) {
+    return assign(value, {
```

Comment:
Or, better yet, use `Object.create` to decorate, then valueOf() can be as simple as `Object.getPrototypeOf(this)`